.env.example

```
@@ -1,45 +1,6 @@
 
-# Database
-DATABASE_URL="postgresql://postgres:password@localhost:54321/incmix"
 
-
-# Service URLs
-
-
-# Authentication
-GOOGLE_REDIRECT_URL="http://localhost:1420/auth/google"
+# Authenticatio GOOGLE_REDIRECT_URL="http://localhost:1420/auth/google"
 GOOGLE_CLIENT_ID="example-google-client-id"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)